src/sentry/snuba/entity_subscription.py

```diff
@@ -375,13 +357,50 @@ def aggregate_query_results(
 class MetricsCountersEntitySubscription(BaseMetricsEntitySubscription):
     entity_key: EntityKey = EntityKey.MetricsCounters
     metric_key: SessionMRI = SessionMRI.SESSION
-    aggregation_func: str = "sum"
+
+    def get_aggregations(self) -> List[List[Optional[str]]]:
+        session_status_crashed = resolve(self.org_id, "crashed")
+        session_status_init = resolve(self.org_id, "init")
+        return [
+            [
+                f"sumIf(value, equals({self.session_status}, {session_status_init}))",
+                None,
+                "count",
```

Comment:
Really nit picking here so feel free to ignore
Why is this called `count` and not `total` ?
Found it to be a bit confusing

Reply:
I agree that `total` would be a better name. I noticed we overwrite the alias with `"count"` here: https://github.com/getsentry/sentry/blob/905088db962c02b2e4f38dde3724850048b25734/src/sentry/incidents/logic.py#L377
so I thought we might as well use it as the original alias.